bundles/org.openhab.binding.evcc/src/main/java/org/openhab/binding/evcc/internal/EvccHandler.java

```diff
@@ -349,6 +391,24 @@ private void createChannelsLoadpoint(int loadpointId) {
         removeChannel(CHANNEL_LOADPOINT_HAS_VEHICLE, channelGroup);
```

Comment:
Please remove all channels you removed or renamed in the according createChannels method.

Reply:
I don't really understand what you mean, sorry. Do you mean removing the unnecessary createChannel(...) lines (I did that already) or do I have to call the removeChannel Method?

Reply:
No problem ;-)
I meant that you need to call removeChannel on the channels you either removed or renamed:

```suggestion
        removeChannel(CHANNEL_LOADPOINT_HAS_VEHICLE, channelGroup);
        removeChannel("minSoC", channelGroup);
        removeChannel("targetEnergy", channelGroup);
        removeChannel("targetSoC", channelGroup);
        removeChannel("targetTime", channelGroup);
        removeChannel("targetTimeEnabled", channelGroup);
```